flake/modules/common/options/misc.nix

```diff
@@ -68,6 +68,17 @@ in {
       '';
     };
 
+    aiRuntime = mkEnableOption ''
+      Whether to enable Firefox AI Runtime
+      This includes stuff like website summaries when hovering over a link
+
+      ::: {.note}
+      LLM inference is done completely on the client, so there is no data
+      or privacy risk.
+      Model itself gets downloaded from model-hub.mozilla.org
+      :::
+    '';
```

Comment:
We'd like to use `aiRuntime.enable` and `aiRuntime.rootURL` here I think. Using `mkENableOption` on non-`.enable` options usually leads to future refactors.